e2eTests/e2eTestPrompts.md

```diff
@@ -328,3 +328,16 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | Tool Name | Test Prompt |
 |:----------|:----------|
 | azmcp-bicepschema-get | How can I use Bicep to create an Azure OpenAI service? |
+
+## Deploy
+| Tool Name | Test Prompt |
+|:----------|:----------|
+| azmcp-deploy-plan-get | Create a plan to deploy this application to azure |
+| azmcp-deploy-iac-rules-get | Show me the rules to generate bicep scripts  |
+| azmcp-deploy-azd-app-log-get | Show me the log of the application deployed by azd  |
```

Comment:
Seems like this should be in the azd tool not here.

Reply:
Tool `azmcp-deploy-plan-get` aims to help developer design the azure service architecture based on the project source code, and then deploy the services using tools `azd` or `az` based on the context. The plan is not just for `azd` command, so we need a separate command group `deploy`.
Tool `azmcp-deploy-iac-rules-get` returns the iac rules for specified azure resources, these rules are defined based on the common errors that caught from our deployment benchmark test result. The rules are not just for `azd` compatible.
Tool `azmcp-deploy-azd-app-log` is for `azd` only, it is supposed to be inside the `azd cli` , we will remove it once `azd cli`  support to get the application log.